tool/site/client-react/src/components/prescription-tracker/prescriptionSearch.tsx

```diff
@@ -39,17 +60,26 @@ const PrescriptionSearch: React.FC<PrescriptionSearchProps> = ({
           />
           <Button onClick={handleSearch}>Search</Button>
           <Button secondary href={baseUrl}>Back</Button>
-        </div>
-        : <div>
-          <Label isPageHeading>Search Results</Label>
+        </>
+        : <>
+          <Label isPageHeading>Found {searchResults.count} Prescription{searchResults.pluralSuffix}</Label>
+          {/* todo: handle multiple prescriptions */}
+          {searchResults.prescriptions
```

Comment:
do we want to handle the case where no prescriptions are found?

Reply:
yeah we will, but think this should be the next pr